fluidkeys/secretreceive.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/fluidkeys/fluidkeys/colour"
+
+	"github.com/fluidkeys/crypto/openpgp"
+	"github.com/fluidkeys/crypto/openpgp/armor"
+	"github.com/fluidkeys/fluidkeys/api"
+	"github.com/fluidkeys/fluidkeys/out"
+	"github.com/fluidkeys/fluidkeys/pgpkey"
+)
+
+func secretReceive() exitCode {
+	out.Print("\n")
+	client := api.NewClient()
+	keys, err := loadPgpKeys()
+
+	if err != nil {
+		printFailed("Couldn't load PGP keys")
+		return 1
+	}
+
+	out.Print(colour.Info("Downloading secrets...") + "\n\n")
+
+	for _, key := range keys {
+		email, err := key.Email()
+		if err != nil {
+			printFailed("Couldn't get email for key " + key.Fingerprint().String() + "\n")
+			continue
+		}
+
+		out.Print("For " + email + ":\n\n")
+		if encryptedSecrets, err := client.ListSecrets(key.Fingerprint()); err != nil {
+			printFailed(err.Error() + "\n")
```

Comment:
i think you'd benefit from `continue` here rather than the subsequent `else` at a new level of indentation

Reply:
Ah. Or bringing out `var encryptedSecrets []v1structs.EncryptedSecret` on the line above

Reply:
But better what you said.

Reply:
👍